geonode/security/models.py

```diff
@@ -17,6 +17,10 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 #########################################################################
+try:
```

Comment:
very small comment, I think that with the django version we have and 2.7 as required python version, this could be opnly import json.